README.md

```diff
@@ -39,7 +54,12 @@ const did = generateDID(key); // Code from ssi-did lib. Returns initial DID Docu
 
 #### KMS Interface
 
-## Relationship to other EWF components
+### Other apps/components
+
+### Switchboard
+
+[Switchboard](https://github.com/energywebfoundation/switchboard-dapp) can be used a UI to initiate and continue a VC-API exchange.
```

Comment:
i am not sure if I understand this sentence correctly. Maybe it should be: "...can be used as a UI..."

Reply:
Thanks!

Reply:
thanks!